### ISSUE TITLE
Please sign the executable

### ISSUE THREAD

**Issue by ale5000-git** [rmyorston/busybox-w32#110]
Sorry to bother you again.
Sorry to bother you again.
I was seeing that there are signature files so I thought that it would be nice if the signature was in the exe itself (in a way that can be seen from explorer).

There is a method of signing on Linux [here](https://stackoverflow.com/questions/18287960/signing-windows-application-on-linux-based-distros).

**Comment by rmyorston** [comment]
I've had a look at this, but I'm not convinced.

@vszakats

**Comment by rmyorston** [comment]
@vszakats (who has raised some busybox-w32 issues in the past) has a nice [script](https://gist.github.com/vszakats/7ef9e86506f5add961bae0412ecbe696) to handle certificate management.

I've tried this out but I'm not convinced.  While it's nice that the signature can be included with the binary:

- a simple signature adds 3.5K to the size of the executable, with a timestamp it adds 8.5K;
- a self-signed certificate results in unpleasant warnings in the explorer properties dialog;
- I'm pretty sure more people will never bother to check the signature.

**Comment by vszakats** [comment]
I agree that self-signed signatures aren't really useful for most cases. A signature accepted by default Windows installations will almost certainly cost money though. At least I couldn't find a free option as of mid last year.

**Comment by ale5000-git** [comment]
@rmyorston: I'm asking it because in addition of being visible in explorer which is nice, in the future I will also add signature verification in my scripts.

**Comment by Ronsor** [comment]
Code signing certs cost money though.
